Packs/Gatewatcher/Integrations/Gcenterv102/Gcenterv102.yml

```diff
@@ -0,0 +1,445 @@
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.created
+      description: ''
+      type: Date
+    - contextPath: GCenter.Gscan.Powershell.username
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.user_agent
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.ip_address
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.file_name
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.sha256
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.is_clean
+      description: ''
+      type: Boolean
+    - contextPath: GCenter.Gscan.Powershell.is_analysis_successful
+      description: ''
+      type: Boolean
+    - contextPath: GCenter.Gscan.Powershell.status
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Powershell.proba_obfuscated
+      description: ''
+      type: Number
+    - contextPath: GCenter.Gscan.Powershell.analysis_score
+      description: ''
+      type: Number
+    - contextPath: GCenter.Gscan.Powershell.is_whiteblack_listed
+      description: ''
+      type: Boolean
+  - name: gw-send-shellcode
+    description: Send shellcode
+    arguments:
+    - name: name
+      required: true
+      description: Filename
+    - name: content
+      required: true
+      description: File content
+    - name: deep
+      required: true
+      description: Deep scan
+      defaultValue: false
+    - name: timeout
+      required: true
+      description: Deep scan timeout
+      defaultValue: 120
+    outputs:
+    - contextPath: GCenter.Gscan.Shellcode.id
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.created
+      description: ''
+      type: Date
+    - contextPath: GCenter.Gscan.Shellcode.username
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.user_agent
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.ip_address
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.file_name
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.sha256
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.is_clean
+      description: ''
+      type: Boolean
+    - contextPath: GCenter.Gscan.Shellcode.is_analysis_successful
+      description: ''
+      type: Boolean
+    - contextPath: GCenter.Gscan.Shellcode.status
+      description: ''
+      type: String
+    - contextPath: GCenter.Gscan.Shellcode.architecture
+      description: ''
+      type: Unknown
+    - contextPath: GCenter.Gscan.Shellcode.is_whiteblack_listed
+      description: ''
+      type: Boolean
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.4.29342
+fromversion: 5.5.0
```

Comment:
Please change the 'fromversion' field to 6.2.0 - that's our default for new content.

Reply:
done